README.rst

```diff
@@ -87,6 +87,7 @@ optional arguments:
                         attempt to verify package
   -p, --prep-only       Only perform preparatory work on package
   --non_interactive     Disable interactive mode for package verification
+  --infile              Additional input file to contribute to specfile creation
```

Comment:
Should be `--infile INFILE`

Should also mention that `INFILE` can be a directory containing bitbake files, a single file, or a url to a file